build/install_xla_in_source_tree.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import shutil
+import argparse
+
+from bazel_tools.tools.python.runfiles import runfiles
+
+parser = argparse.ArgumentParser()
+parser.add_argument("target")
+args = parser.parse_args()
+
+r = runfiles.Create()
+
+def _is_windows():
+    return sys.platform.startswith("win32")
+
+def _copy_so(src_file, dst_dir):
+    src_filename = os.path.basename(src_file)
+    if _is_windows() and src_filename.endswith(".so"):
+        dst_filename = src_filename[:-3] + ".pyd"
+    else:
+        dst_filename = src_filename
+    dst_file = os.path.join(dst_dir, dst_filename)
+    shutil.copyfile(src_file, dst_file)
+
+def _copy_normal(src_file, dst_dir):
+    src_filename = os.path.basename(src_file)
+    dst_file = os.path.join(dst_dir, src_filename)
+    shutil.copyfile(src_file, dst_file)
+
+
+def copy(src_file, dst_dir=os.path.join(args.target, "jaxlib")):
+    # NOTE: this might be useful for debugging on Windows to inspect into the
+    #  Temp dir structure. So I am leaving it here, for now.
+    # try:
+    #     src_file.endswith(".so")
+    # except:
+    #     import traceback
+    #     import time
+    #     traceback.print_exc()
+    #     time.sleep(10000)
+
+    if src_file.endswith(".so"):
+        _copy_so(src_file, dst_dir)
+    else:
+        _copy_normal(src_file, dst_dir)
+
+def patch_copy_xla_client_py(dst_dir=os.path.join(args.target, "jaxlib")):
+    with open(r.Rlocation("org_tensorflow/tensorflow/compiler/xla/python/xla_client.py")) as f:
+        src = f.read()
+        src = src.replace("from tensorflow.compiler.xla.python import xla_extension as _xla", "from . import xla_extension as _xla")

```

Comment:
Not for this PR, but I'm reminded that we can probably avoid this by just changing the code in the TF tree.